CLAUDE.md

```
@@ -0,0 +1,227 @@
+| Boss HP | 12-20 | constants.ts WAVES |
+| Ability cooldown | 420ms | game-logic.ts (hardcoded) |
+| Nuke cooldown | 12000ms | game-logic.ts (hardcoded) |
+| Director build rate | 0.03-0.08/s | director.ts |
+| Boss burst count | 3-7 | boss-ai.ts |
+| Boss sweep count | 5-9 | boss-ai.ts |
+| Boss spiral count | 8-14 | boss-ai.ts |
+| Evaluator biases | 0.5-1.2 | boss-ai.ts |
+
+## Completed Work (PR #45)
+
+### Commits pushed to `claude/fix-outstanding-issues-jwlJ3`:
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)